test/rails_app/config/application.rb

```diff
@@ -31,8 +31,10 @@ class Application < Rails::Application
     # config.assets.enabled = false
 
     config.action_mailer.default_url_options = { host: "localhost", port: 3000 }
-
-    if DEVISE_ORM == :active_record && (Rails::VERSION::MAJOR >= 4 && Rails::VERSION::MINOR >= 2)
+    rails_version = Gem::Version.new(Rails.version)
```

Comment:
Would you mind explaining why do we need this change?

Reply:
`raise_in_transactional_callbacks` was deprecated in rails 5.0 and removed in 5.2 https://github.com/rails/rails/commit/8029f779b8a1dd9848fee0b7967c2e0849bf6e07

Reply:
I see. The strange thing is that some specs are failing on CI with an error that seems to be related to this
```
`method_missing': undefined method `raise_in_transactional_callbacks=' for ActiveRecord::Base:Class
```
Could you check those errors?

Reply:
This is now fixed.

Reply:
@jcoyne is it fixed on `master`? I still encounter this error